src/control/cmd/daos_agent/start.go

```diff
@@ -117,6 +120,10 @@ func (cmd *startCmd) Execute(_ []string) error {
 
 	cmd.Debugf("startup complete in %s", time.Since(startedAt))
 	cmd.Infof("%s (pid %d) listening on %s", versionString(), os.Getpid(), sockPath)
+	if err := systemd.Ready(); err != nil && err != systemd.ErrSdNotifyNoSocket {
```

Comment:
if no socket exists don't we want to notify or at least debug log that the notify wasn't sent?

Reply:
We want to support the non-systemd use case. It's true that there could be a bug where the agent was launched by systemd but there was no socket, but this seems like a pretty rare situation and not worth the log spam in the non-systemd use case.